pkg/metrics/controller.go

```diff
@@ -258,11 +271,60 @@ func (c *Controller) recordGameServerStatusChanges(old, next interface{}) {
 	if newGs.Status.State != oldGs.Status.State {
 		fleetName := newGs.Labels[agonesv1.FleetNameLabel]
 		if fleetName == "" {
-			fleetName = "none"
+			fleetName = defaultFleetTag
 		}
 		recordWithTags(context.Background(), []tag.Mutator{tag.Upsert(keyType, string(newGs.Status.State)),
 			tag.Upsert(keyFleetName, fleetName)}, gameServerTotalStats.M(1))
+
+		// Calculate the duration of the current state
+		duration, err := c.calcDuration(oldGs, newGs)
+		if err != nil {
+			c.logger.Debug(err.Error())
```

Comment:
I would think warning is better suited for this. I would also wrap the message to add more meat about the context.

Reply:
WDYT ?

Reply:
I would agree, Warn() would be better, unless we are specifically trying to ignore any error here?